lib/widgets/loading_indicator.dart

```diff
@@ -9,3 +9,11 @@ class LoadingIndicator extends StatelessWidget {
     );
   }
 }
+
+Future<void> showProgressIndicator(BuildContext context) {
```

Comment:
`topcontext` -> `topContext` ;)

Reply:
I marked wrong lines, but you surely understand what I mean

Reply:
got it, will get it done asap :)

Reply:
> `topcontext` -> `topContext` ;)

good catch 👍

Reply:
Done